client/src/get-ticket/components/home.tsx

```diff
@@ -1,14 +1,45 @@
-import { useState } from 'react'
+import { useEffect } from 'react'
 import Button from 'react-bootstrap/Button';
 import '../get-ticket.css';
 import { useNavigate } from 'react-router-dom';
+import API from '../../API/API.tsx';
 
```

Comment:
You may need to define this interfaces here as the folowing:

interface Service {
  ServiceID: number;
  ServiceName: string;
}

interface Props {
  ticket: Ticket;
  setTicket: React.Dispatch<React.SetStateAction<Ticket>>;
}